pkg/model/components/apiserver.go

```diff
@@ -198,6 +198,12 @@ func (b *KubeAPIServerOptionsBuilder) BuildOptions(o interface{}) error {
 		}
 	}
 
+	if b.IsKubernetesLT("1.20") && clusterSpec.ServiceAccountIssuerDiscovery != nil && fi.BoolValue(&clusterSpec.ServiceAccountIssuerDiscovery.EnableAWSOIDCProvider) {
```

Comment:
Do we need a minimum version here too, in case we want to cherry-pick this to other release branches?

Reply:
I added some validation to cover this case.